src/content/docs/apis/nerdgraph/examples/async-queries-nrql-tutorial.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "NerdGraph tutorial: Run NRQL queries asynchronously"
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+metaDescription: Use New Relic NerdGraph to make asynchronous NRQL queries that have longer query duration limits.
+---
+
+You can use our NerdGraph API to run an asynchronous NRQL query. This is useful for running queries that have less risk of being interrupted by browser or HTTP connection timeout limits or other issues. It's also especially applicable for running queries that may take a long time to complete.
```

Comment:
```suggestion
Our NerdGraph API can run an asynchronous NRQL query with `async: true`. If you have a query that may take a long time to complete, using an asynchronous NRQL query introduces less risk of query interruption by browser or HTTP connection timeout limits or other issues.
```

Reply:
Curious if "other issues" is necessary -- I'd recommend either omitting it entirely unless it prevents interruption from any kind of issue. e.g., does this preserve the ~integrity of a query if there's infrastructure failure, or is only useful for interruptions? Not sure if that nuance makes sense but it might be worth teasing apart at least the genre of issues it reduces risk for